### DISCUSSION TITLE
[建议] 文档查看器,归档管理器,浏览器

### DISCUSSION THREAD

**Discussion by LTreeshu** [linuxdeepin/developer-center#3120] [Features Request & Ideas | 特性请求 & 头脑风暴]
# #

**Comment by BLumia** [comment]
hi, 建议类反馈我这边帮您挪到讨论板，如果是 bug 再报到 issue 面板哦

**Reply by LTreeshu** [reply]
我不想提这个issue了，你们傻逼要求太多了，爱听不听

**Reply by BLumia** [reply]
issue 板块是跟踪 bug 的，我们用 discussion 是为了更高效率的处理需求和 bug。

我好像是第一次处理你的 issue。大家谁也不欠谁的，不想提可以不提，带脏字带情绪不仅没什么意思而且无法产生任何有意义的讨论👎

**Reply by LTreeshu** [reply]
嗯嗯，对不起，有点激动了，我郑重的向你道歉，希望你能接受

**Reply by BLumia** [reply]
没关系的，虽然不清楚你这边发生了什么。我这边留意到你其实还有一些一两年前汇报的 issue 也是最近才得到我们响应，不清楚是不是因为这个对我们感到有点失望。

我们开源社区中心是去年才成立的，今年才逐步开始慢慢恢复开源社区相关的事物处理流程，之前沉积的问题现在才响应还请见谅。这个相关其实有个说明性质的 issue 可以参阅（在 https://github.com/linuxdeepin/developer-center/issues/2830 ），如果有建议意见并且有意见提的话还是欢迎大家提起的。我在推进相关的工作，所以如果能帮上忙的话我可以尽量予以协助。

无论参与与否，希望你能理解 :)

（ps. 如果只是因为不再使用 deepin 所以不再希望被之前的帖子打扰到的话，可以用右侧的 Unsubscribe 按钮来取消订阅，就不会收到消息推送了。我看你这边反馈的很多问题其实确实存在并且我也是认同的，甚至有一些是已经采纳(标了 accepted 标签)后续会进行修改的。所以如果愿意的话，帖子仍然保留其实还是有价值的）

**Reply by LTreeshu** [reply]
嗯嗯，好的，辛苦了
